### PR TITLE
fix: #CRRE-499 fix filter search and do rename

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
Renamed to avoid confusion.
Fix on search bar
Fix nversion between consumable and credit

## Checklist tests
Repeat checklist #63 

## Issue ticket number and link
[CRRE-499](https://jira.support-ent.fr/browse/CRRE-499)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/template/validator/order-waiting.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div ng-controller="waitingValidatorOrderController" class="crre-table">
     <div class="crre-order-waiting-header ten">
-        <order-filter-validator class="flex-one" on-search="search" filter="filterOrder" user-list="users" type-campaign-list="type_campaign"></order-filter-validator>
+        <order-waiting-filter-validator class="flex-one" on-search="search" filter="filterOrder" user-list="users" type-campaign-list="type_campaign"></order-waiting-filter-validator>
```
Comment (BastienBenard) [inline src/main/resources/public/template/validator/order-waiting.html:3]: Renommage

File: src/main/resources/public/ts/directives/index.ts [hunk 2]
```diff
@@ -6,4 +6,4 @@ export * from './infinite-scroll';
 export * from './loader';
 export * from './confirm-generate-library/confirm-generate-library.directive';
 export * from './order-waiting-price-display/order-waiting-price-display.directive';
-export * from './order-filter-validator/order-filter-validator.directive';
\ No newline at end of file
+export * from './order-waiting-filter-validator/order-waiting-filter-validator.directive';
```
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/index.ts]: Renommage

File: src/main/resources/public/ts/directives/order-waiting-filter-validator/order-waiting-filter-validator.directive.ts
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/order-waiting-filter-validator/order-waiting-filter-validator.directive.ts:1]: Renommage pour éviter la confusion

File: src/main/resources/public/ts/directives/order-waiting-filter-validator/order-waiting-filter-validator.html [hunk 3]
```diff
@@ -38,15 +38,15 @@
     <multi-combo title="[[vm.lang.translate('Teachers')]]"
                  combo-model="vm.userList"
                  search-on="user_name" order-by="user_name"
-                 labels="vm.comboLabels" filtered-model="vm.filter.users"
+                 labels="vm.comboLabels" filtered-model="vm.filter.userList"
```
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/order-waiting-filter-validator/order-waiting-filter-validator.html:41]: Renommage

File: src/main/resources/public/ts/directives/order-waiting-filter-validator/order-waiting-filter-validator.html [hunk 4]
```diff
@@ -38,15 +38,15 @@
...
                  selection-event="vm.applyFilter()"
                  deselection-event="vm.applyFilter()"
                  class="two inline-block horizontal-spacing no-border">
     </multi-combo>
     <multi-combo title="[[vm.lang.translate('crre.type.campaign')]]"
                  combo-model="vm.typeCampaignList"
                  search-on="name" order-by="name"
-                 labels="vm.comboLabels" filtered-model="vm.filter.typeCampaign"
+                 labels="vm.comboLabels" filtered-model="vm.filter.typeCampaignList"
```
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/order-waiting-filter-validator/order-waiting-filter-validator.html:49]: Renommage

File: src/main/resources/public/ts/model/ValidatorOrderWaitingFilter.ts [hunk 5]
```diff
@@ -6,17 +6,17 @@ export class ValidatorOrderWaitingFilter {
     startDate;
     endDate;
     queryName: string;
-    users: Array<UserModel>;
-    typeCampaign: Array<Campaign>;
-    filterChoiceCorrelation: Array<string>;
+    userList: Array<UserModel>;
+    typeCampaignList: Array<Campaign>;
```
Comment (BastienBenard) [inline src/main/resources/public/ts/model/ValidatorOrderWaitingFilter.ts:10]: Renommage

File: src/main/resources/public/ts/model/ValidatorOrderWaitingFilter.ts [hunk 6]
```diff
@@ -6,17 +6,17 @@ export class ValidatorOrderWaitingFilter {
...
-    constructor(filterChoiceCorrelation: Array<string>) {
+    constructor() {
         this.startDate = moment().add(-6, 'months')._d;
         this.endDate = moment()._d;
         this.queryName = "";
-        this.users = [];
-        this.typeCampaign = [];
-        this.filterChoiceCorrelation = filterChoiceCorrelation;
+        this.userList = [];
+        this.typeCampaignList = [];
```
Comment (BastienBenard) [inline src/main/resources/public/ts/model/ValidatorOrderWaitingFilter.ts:19]: Renommage

File: src/main/resources/public/ts/directives/order-waiting-price-display/order-waiting-price-display.directive.ts [hunk 7]
```diff
@@ -64,7 +64,7 @@ class Controller implements IViewModel {
 
     totalSelectedPriceConsumable(): number {
         if (this.allOrdersSelected) {
-            return this.amountTotal.credit;
+            return this.amountTotal.consumable_credit;
```
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/order-waiting-price-display/order-waiting-price-display.directive.ts:67]: Inversion entre consommable et crédit

File: src/main/resources/public/ts/directives/order-waiting-price-display/order-waiting-price-display.directive.ts [hunk 8]
```diff
@@ -76,7 +76,7 @@ class Controller implements IViewModel {
 
     totalSelectedPrice(): number {
         if (this.allOrdersSelected) {
-            return this.amountTotal.consumable_credit;
+            return this.amountTotal.credit;
```
Comment (BastienBenard) [inline src/main/resources/public/ts/directives/order-waiting-price-display/order-waiting-price-display.directive.ts:79]: Inversion entre consommable et crédit

**Review by sjollois — APPROVED** [review]

**Review by JinDouang — APPROVED** [review]

**Pull request merged**